### PR TITLE
Rename incorrect names to correct names

### PR DESCRIPTION
**Pull request by reddikih**
Renamed `Scalar DL` and `Scalar DLT` to `ScalarDL` to use correct name.
And also I renamed `Scalar Client SDK` to `Scalar Java Client SDK` both in documents and the image file.

NOTE:
This repository seems old. For example, according to the [manual/README.md](https://github.com/scalar-labs/getting-started-with-scalardl/blob/master/manual/README.md#connect-to-scalar-dl-sandbox), the Scalar DL Sandbox is required but the URL indicated to the Scalar DL Sandbox is broken (maybe removed?).

However, this tutorial is still linked from [scalardl/docs/tutorial](https://github.com/scalar-labs/scalardl/tree/master/docs#tutorials). So I think it is better to update this tutorial to support the latest version of the ScalarDL and environment (i.e. use Docker container image).

Anyway, I think it should be done as another task (I will do it). 

What do you think? @feeblefakie @supl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,28 +12,28 @@
         - [Unit Test](#unit-test)
           - [Mock the ledger object](#mock-the-ledger-object)
           - [Assert the result of execution](#assert-the-result-of-execution)
-        - [Use Scalar DL Emulator](#Use-Scalar-DL-Emulator)
+        - [Use ScalarDL Emulator](#Use-Scalar-DL-Emulator)
```
Comment (feeblefakie) [inline README.md:15]: Emulator is not available any more, please delete the section.

File: manual/README.md [hunk 2]
```diff
@@ -27,16 +27,16 @@ java -jar am-1.0.jar <command> [<command options>|<command arguments>]
...
-`init` command is used to generate a config file `client.properties` so that the other commands can use it to connect to a Scalar DL network afterwards.
+`init` command is used to generate a config file `client.properties` so that the other commands can use it to connect to a ScalarDL network afterwards.
 
 ```
 java -jar am-1.0.jar init [-h] [-tls] [-credential=<credential>] [-host=<host>] [-p=<port>] private-key certificate
 ```
 
-#### Connect to Scalar DL Sandbox
-We can test this application with Scalar DL Sandbox provided by Scalar, Inc.
+#### Connect to ScalarDL Sandbox
```
Comment (feeblefakie) [inline manual/README.md:36]: We don't have Sandbox environment any more. Please delete the section.

File: manual/use-scalar-dl-sandbox.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
-# Apply for using Scalar DL Sandbox
+# Apply for using ScalarDL Sandbox
```
Comment (feeblefakie) [inline manual/use-scalar-dl-sandbox.md:1]: Ditto.

File: README.md [hunk 4]
```diff
@@ -12,28 +12,28 @@
...
 
-In this tutorial we introduce how to implement a simple asset management application on Scalar DLT solution.
+In this tutorial we introduce how to implement a simple asset management application on ScalarDL solution.
 If you haven't tried this application yet, please refer to [here](./manual/README.md).
 
 ## Overview
 
-Scalar DLT is a distributed ledger platform.
-A Scalar DL network exposes a key-value semantics called ledger to clients.
+ScalarDL is a distributed ledger platform.
```
Comment (feeblefakie) [inline README.md:27]: Can you update the description to the current one? (I think we need a consistent description)

**Review by feeblefakie — COMMENTED** [review]
Thank you!
Left several comments.
I think we should update the doc to make it work with the latest version and environments in addition to updating wording, otherwise, this doc is not really usable for users.
Or, we just archive the repo.

**Comment by feeblefakie** [comment]
Maybe, it's better to remove/archive it. @supl What do you think?

**Comment by supl** [comment]
> Maybe, it's better to remove/archive it. @supl What do you think?

@reddikih 
@feeblefakie 

Thanks for your asking.

Personally, I would recommend doing this:

1. Revise the document and source code to align with the latest ScalarDL version (The sandbox part could be replaced by scalardl-samples as we mentioned in the [How to Use](https://github.com/scalar-labs/scalardl/blob/master/docs/index.md) in the scalardl repository)
2. Migrate this repository to a sub-folder in the scalardl repository
3. Archive this repository

The reasons for doing this:
1. We still need a getting-started to give the ScalarDL developers an overview. Otherwise, I feel something is missing when I try to read the ScalarDL document page [here](https://github.com/scalar-labs/scalardl/blob/master/docs/index.md). (Image we remove the "getting-started" item from the "How to Use" section)
2. This repository was originally done by Indetail so it is independent of the scalardl repository, the one we gather the documents for ScalarDL. Maybe it's a good chance to migrate it.

**Comment by feeblefakie** [comment]
@reddikih @supl  Let's archive this repo since it's too old.
But, I think we might need to create something similar in the near future. Let's discuss it later.

**Pull request closed without merging**